webrender_api/src/display_list.rs

```diff
@@ -319,15 +322,20 @@ impl<'a, 'b> DisplayItemRef<'a, 'b> {
     }
 
     pub fn rect(&self) -> LayoutRect {
-        self.iter.cur_item.rect
+        self.iter.cur_item.info.rect
     }
 
-    pub fn local_clip(&self) -> &LocalClip {
-        &self.iter.cur_item.local_clip
+    pub fn get_layer_primitive_info(&self, offset: &LayoutVector2D) -> LayerPrimitiveInfo {
+        let info = self.iter.cur_item.info;
+        LayerPrimitiveInfo {
+            rect: info.rect.translate(&offset),
+            local_clip: info.local_clip.map(|clip| clip.create_with_offset(offset)),
+            is_backface_visible: info.is_backface_visible,
+        }
     }
 
-    pub fn local_clip_with_offset(&self, offset: &LayoutVector2D) -> LocalClip {
-        self.iter.cur_item.local_clip.create_with_offset(offset)
+    pub fn local_clip(&self) -> Option<LocalClip> {
```

Comment:
In all but one case this function result is unwrapped right away, and it used to return `LocalClip` directly. Would be simpler to keep it this way and change less code, I think.